pyproject.toml

```diff
@@ -187,7 +187,7 @@ docs-gettext-pot = { "cwd" = "docs", "shell" = "sphinx-build -b gettext . locale
 clean = { "shell" = "rm -rf dist build _packaging/dist _packaging/build _packaging/windows/file_version_info.txt docs/_build" }
 package = { "cwd" = "_packaging", shell = "pyinstaller -y gaphor.spec"}
 win-installer = { "script" = "_packaging.windows.build-win-installer:main" }
-gettext-pot = "pybabel extract -o po/gaphor.pot -F po/babel.ini -k i18nize gaphor"
+gettext-pot = "pybabel extract --omit-header --project Gaphor -o po/gaphor.pot -F po/babel.ini -k i18nize gaphor"
```

Comment:
Why omit the header?

Reply:
Hey @amolenaar, I thought the build was failing since there was license info in the .pot file. But looking again maybe not. If we are going to keep the header - we should update the project name and other info. Do you think the header is important?

Reply:
I went ahead and dropped this commit, I think we can make this part of another PR if we try to remove or update the po headers.

Reply:
Indeed. Let's change that in a future commit.